File: VB/E95/App_Code/GridData.vb
Option Infer On

Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Public Class GridData
    Public Sub New()

    End Sub
    Public Sub New(ByVal employeeID As Integer, ByVal firstName As String, ByVal lastName As String, ByVal photo() As Byte)
        Me.EmployeeID = employeeID
        Me.FirstName = firstName
        Me.LastName = lastName
        Me.Photo = photo
    End Sub
    Public Property EmployeeID() As Integer
    Public Property FirstName() As String
    Public Property LastName() As String
    Public Property Photo() As Byte()
End Class

File: CS/E95/App_Code/GridData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class GridData {
    public GridData() {

    }
    public GridData(int employeeID, string firstName, string lastName, Byte[] photo) {
        EmployeeID = employeeID;
        FirstName = firstName;
        LastName = lastName;
        Photo = photo;
    }
    public int EmployeeID { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public Byte[] Photo { get; set; }
}